ocrs/src/layout_analysis/empty_rects.rs

```diff
@@ -64,7 +64,7 @@ where
             queue.push(Partition {
                 score: score(boundary),
                 boundary,
-                obstacles: obstacles.to_vec(),
+                obstacles: obstacles.clone(),
```

Comment:
Looking into the code, obstacles is already made into a vec above 
line 57  `let mut obstacles = obstacles.to_vec(); `
Since is not consumed elsewhere later I think we can just use the shorthand initialization without `.clone()` or `to_vec()`

it would look like this
```
queue.push(Partition {
                score: score(boundary),
                boundary,
                obstacles,
            });
```
do you still prefer the original over this?

Reply:
You're right. The clone looks redundant. This could be made clearer by moving initialization of it inside the `!boundary.is_empty()` check.